### DISCUSSION TITLE
Reverse Auth/Session?

### DISCUSSION THREAD

**Discussion by nacoleman** [VisualReCode/Cocoon#6] [Q&A]
Hi Mark, haven’t had time to watch all the streams yet. Was just wondering what happens when we need this to work in reverse? What I mean is... if I wanted to host/convert the login/auth pages to the .net core facade application rather than the legacy app.. or would that have to be the last thing to port over?

**Comment by TheRubble** [comment]
I'd suggest that you leave the authentication pages to the end. I've done a few ports now and it's worked well, I'd also suggest making the relationship two way between the host and cocooned application could easily become problematic the single direction currently makes the thought process easier.

For authentication what I've done, is migrate the legacy application to port data into Auth0 and slowly move the use the users and data across over time. After a sensible period I'll force old users through a password reset flow and hopefully all will be good.
